explorer/src/Stories/Badge.elm

```diff
@@ -45,4 +45,28 @@ stories =
                     ]
           , {}
           )
+        , ( "Number no icon"
+          , \_ ->
+                Html.div [ css [ displayFlex ] ]
+                    [ Badge.Number 4 Nothing |> Badge.view [] [] ]
+          , {}
+          )
+        , ( "Number with max char count"
+          , \_ ->
+                Html.div [ css [ displayFlex ] ]
+                    [ Badge.Number 110 (Just 2) |> Badge.view [] [] ]
+          , {}
+          )
+        , ( "Dot"
+          , \_ ->
+                Html.div [ css [ displayFlex ] ]
+                    [ Badge.Dot |> Badge.view [] [ Icons.calendar [ css [ width (rem 2) |> Css.important ] ] ] ]
+          , {}
+          )
+        , ( "Dot no icon"
+          , \_ ->
+                Html.div [ css [ displayFlex ] ]
+                    [ Badge.Dot |> Badge.view [] [] ]
+          , {}
+          )
```

Comment:
Do we need this dot variant? Best if we could keep it simple and only have one generic variant

Reply:
We have the design here. The dot is the one we need to use when we don't have the number. I couldn't find the generic variant in design lib and Line too didn't know about this one. 
![image](https://github.com/user-attachments/assets/a4b2bb4f-8f47-44ea-93a1-09fabe7d4060)

Reply:
If we are not using this generic one then it should be replaced to dot. What you think?

Reply:
Yes I agree, challenge Line about replacing the old generic one rather than adding a new